src/BudgetManager/BudgetManager.API/Controllers/TransactionController.cs

```diff
@@ -28,6 +28,13 @@ public async Task<IActionResult> GetTransactionById(Guid id, CancellationToken c
             return response == null ? NotFound() : Ok(response);
         }
 
+        [HttpGet("{userId}/{opterationType}")]
+        public async Task<IActionResult> GetTransactionsByOperation(Guid userId, string operationType, CancellationToken cancellationToken)
```

Comment:
I suggest to create a separate DTO which will include userId and operationType and in my opinion it should be not string but Enum type. It is Frontend's responsibility to map enum types in the UI and send us as an enum value not a string